portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -53,6 +53,17 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       comments.add(comment);
       response.sendRedirect("/index.html");  
```

Comment:
You should return early here, otherwise you'll still store an empty comment

Reply:
What do you mean exactly? Return what early?

Reply:
just return void (i.e. "return;").